.circleci/config.yml

```diff
@@ -36,16 +36,29 @@ parameters:
     default: false
 
 commands:
-  install-dependencies:
+  install-dependencies-unix:
     steps:
       - restore_cache:
           keys:
-            - yarn-v2-{{ checksum "yarn.lock" }}
+            - yarn-v2-unix-{{ checksum "yarn.lock" }}
       - run:
           name: Yarn Install
           command: yarn
       - save_cache:
-          key: yarn-v2-{{ checksum "yarn.lock" }}
+          key: yarn-v2-unix-{{ checksum "yarn.lock" }}
+          paths:
+            - .yarn/cache
+
+  install-dependencies-macos:
+    steps:
+      - restore_cache:
+          keys:
+            - yarn-v2-macos-{{ checksum "yarn.lock" }}
```

Comment:
Instead of duplicating the job, this could have been a `prefix` parameter.

Reply:
Yeah, that could also work. It moves the duplication of the prefix/machine to the call site... But it might be ok. I'm changing it in #843. Lmk what you think!